src/style.css

```diff
@@ -1 +1,80 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 500;
+  font-style: normal;
+  text-transform: uppercase;
+  background: #e5e5e5;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+header {
```

Comment:
use class selectors instead of tag ones

Reply:
fix it https://github.com/mate-academy/layout_moyo-header/pull/2832/files#r1134143794